<launch>
   <!-- launch video stream -->
   <include file="$(find video_stream_opencv)/launch/camera.launch" >
        <!-- node name and ros graph name -->
        <arg name="camera_name" value="camera_1" />
        <!-- means video device 0, /dev/video0 -->
        <arg name="video_stream_provider" value="0" />
   </include>

</launch>
